src/main/java/com/charu/Tasks.java

```diff
@@ -0,0 +1,51 @@
+package com.charu;
+
+class Tasks{
+
+
+
+    private int id;
```

Comment:
Rename it with taskId, taskName, taskStatus

Reply:
addressed